src/transformers/models/longt5/modeling_longt5.py

```diff
@@ -1349,10 +1350,11 @@ def _shift_right(self, input_ids):
         decoder_start_token_id = self.config.decoder_start_token_id
         pad_token_id = self.config.pad_token_id
 
-        assert decoder_start_token_id is not None, (
-            "self.model.config.decoder_start_token_id has to be defined. In LongT5 it is usually set to the pad_token_id."
-            " See LongT5 docs for more information"
-        )
+        if decoder_start_token_id is None:
+            raise ValueError(
+                "self.model.config.decoder_start_token_id has to be defined. In LongT5 it is usually set to the pad_token_id."
+                " See LongT5 docs for more information"
+            )
```

Comment:
This has gone a bit weird with the formatting - shall we put everything onto one line and then let `ruff` fix it for us by calling `make style`?
```suggestion
            raise ValueError("self.model.config.decoder_start_token_id has to be defined. In LongT5 it is usually set to the pad_token_id. See LongT5 docs for more information")
```

Reply:
Ok, trying now.